pac4j-saml/src/main/java/org/pac4j/saml/sso/impl/SAML2DefaultResponseValidator.java

```diff
@@ -681,9 +681,9 @@ protected final void validateSignature(final Signature signature, final String i
     }
 
     private boolean isDateValid(final DateTime issueInstant, final int interval) {
-        final long now = System.currentTimeMillis();
-        return issueInstant.isBefore(now + this.acceptedSkew * 1000)
-                && issueInstant.isAfter(now - (this.acceptedSkew + interval) * 1000);
+        final DateTime before =  DateTime.now().plusSeconds(acceptedSkew);
```

Comment:
This basically looks at the system property to find the timezone, then looks at the TimeZone class of Java to get the default timezone of the system.